src/commands/community/nft/recent.ts

```diff
@@ -12,7 +12,7 @@ async function composeNFTListEmbed(msg: Message, pageIdx: number) {
     size: 16,
   })
 
-  if (!data || !data.length) {
+  if (!data?.data || !data?.data.length) {
```

Comment:
chỗ này ở trên đã destructure ra rồi mà em, anh nghĩ đúng sẽ là, có gì chạy local test lại nha em
```suggestion
  if (!data || (Array.isArray(data) && data.length === 0)) {
```